### DISCUSSION TITLE
Why does anaconda *ALWAYS* end with "Press ENTER to quit"?

### DISCUSSION THREAD

**Discussion by edgarm1964** [rhinstaller/anaconda#4685] [General]
LS,

  title says it all: Why does anaconda *ALWAYS* end with "Press ENTER to quit" when doing an UEFI install? Even in cmdline mode? This is really annoying as it prohibits unattended installs...

Attached are the kickstart and grub.cfg files used.

Kind regards, Edgar.
[grub.cfg.txt](https://github.com/rhinstaller/anaconda/files/11172644/grub.cfg.txt)
[minimal-37.ks.txt](https://github.com/rhinstaller/anaconda/files/11172646/minimal-37.ks.txt)

These files are downloaded using PXE.

**Comment by mikhailnov** [comment]
because you did not write "reboot" in your kickstart ))

**Reply by edgarm1964** [reply]
I removed too  many lines... But, even with reboot or halt it will end with "Installation completed. Press ENTER to quit" So, no, reboot nor halt work.

Sorry, anaconda version 37.12.6
